src/providers/github/Github.ts

```diff
@@ -48,14 +48,22 @@ export class Github {
   // eslint-disable-next-line @typescript-eslint/explicit-module-boundary-types
   async getRepo() {
     try {
-      return await this.octokit.repos.get({
-        owner: this.owner,
-        repo: this.repo
-      });
+      const repoResponse = await this.octokit.request(
+        "GET /repos/{owner}/{repo}",
+        {
+          owner: this.owner,
+          repo: this.repo,
+          headers: {
+            "X-GitHub-Api-Version": "2022-11-28"
+          }
+        }
+      );
+
+      return repoResponse.data;
     } catch (e) {
-      const repoSlug = `${this.owner}/${this.repo}`;
-      if (e.status === "404") throw Error(`Repo does not exist: ${repoSlug}`);
-      e.message = `Error verifying repo ${repoSlug}: ${e.message}`;
+      if (e.status === 404 || e.status === "404")
```

Comment:
Taking advantage of oktokit types you may stop from using this approach. See https://www.npmjs.com/package/@octokit/types

Reply:
Improved